app/views/forms/_comments.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= form_with(model: [locals[:comment], locals[:comment].current_user.comments.build],
+              url: user_post_comments_path(locals[:comment].current_user, locals[:post_id])) do |form| %>
+  <%= form.text_area :text, size: "80x5", placeholder: 'Type your comments here!' %>
+  <%= form.submit 'Publish', size: "40x5", class: 'publish-button' %>
+<% end %>
```

Comment:
- [ ] Kindly fix the error which shows up when a user wants to create a post. Since I failed to create a post due to an error, I couldn't create a comment or a like either.


![post](https://github.com/techmoves/movesBloc/assets/53124548/9a779459-4ecb-432c-8406-47e94d086c33)